flask_multipass/providers/ldap/providers.py

```diff
@@ -122,7 +122,10 @@ def get_members(self):
                                        **to_unicode(user_data))
                 group_filter = build_group_search_filter({self.ldap_settings['member_of_attr']: {group_dn}}, exact=True)
                 subgroups = list(self.provider._search_groups(group_filter))
-                group_dn = group_dns.send(subgroups)
+                try:
+                    group_dn = group_dns.send(subgroups)
+                except StopIteration:
+                    return
```

Comment:
While it makes no difference right now, `break` might be more correct here (AFAIK the pre-3.7 behavior terminated the loop) - so if someone ever adds code here after the loop, it would still work properly